scripts/config.pl

```diff
@@ -115,8 +115,11 @@
 );
 
 # Things that should be enabled in "full" even if they match @excluded
+# These are all the platform ALT definitions, so that we test them in "full",
+# except MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT which requires a platform_alt.h
+# that we do not provide.
```

Comment:
Should we provide one for testing or would that not be valuable? I'm trying to judge if we should file an issue for plugging this test hole.